sw/host/opentitanlib/src/transport/hyperdebug/spi.rs

```diff
@@ -42,6 +45,26 @@ pub const USB_SPI_REQ_ENABLE_EC: u8 = 3;
 const USB_MAX_SIZE: usize = 64;
```

Comment:
What type of USB device is on hyperdebug?  I'm surprised that this is small like a USB 1.1 endpoint (I'd hoped we'd have some nice big USB2 endpoints in the 512 to 2048 bytes range).

Reply:
Unfortunately, STM32L5 has only USB 1 support, that is 64 byte packets, and we cannot even get double buffered endpoint, which would have allowed the host computer to start sending a second 64 byte packet immediately, while HyperDebug firmware was processing the first one.  (HyperDebug hardware does in theory support double buffering, but requires the use of two endpoint slots, and with four UARTS, as well as SPI, I2C, console and the control endpoint, we do not have a spare.)